test/chunking-form/samples/inline-dynamic-import/inline-conditionally/_expected/es/main.js

```diff
@@ -0,0 +1,13 @@
+if (globalThis.unknown) {
+	Promise.resolve().then(function () { return inlined; }).then(console.log);
+}
+
+console.log('main1');
+
+console.log('inlined');
+const value = 'inlined';
+
+var inlined = /*#__PURE__*/Object.freeze({
+	__proto__: null,
+	value: value
+});
```

Comment:
Ok, it could technically be done by first dynamically importing the static dependencies that are in other chunks and THEN execution the conditionally imported code. But it would become quite a bit bigger than it is now because then we need to think about the whole chunk rendering process, not only the colouring algorithm.